recipe/conda_build_config.yaml

```diff
@@ -0,0 +1,4 @@
+# recipe/conda_build_config.yaml
+astropy_outputs:
+  - all    # [linux64 and py310]
```

Comment:
Ooof, i was gravely mistaken: this is the file where `py310` and friends would _come from_, so can't be referenced here (or rather, will always evaluate to `None`).

`linux64` _could_ be used, but looking across the craziverse of magic selectors, it looks like this could go _without_ selectors:

```yaml
astropy_outputs:
  - all
  - base
```

... and then in the recipe (perhaps with an explanatory comment):

```yaml
# build this exactly once, on the oldest supported python, to avoid migration issues
{% if astropy_outputs == "all" and py==310 and linux64 %}
- name: astropy
  build:
    noarch: generic
  requirements:
    run:
      - {{ pin_subpackage("astroy-base", max_pin="x.x.x") }} 
      # ...all the stuff
{% endif %}
```

Reply:
Gah, this will still explode. It needs another `skip: true` somewhere. Let me try some local rerendering...

Reply:
My local env is misbehaving unfortunately, so I am relying on the CI.